msg_helpers.go

```diff
@@ -683,6 +683,10 @@ func packDataAplPrefix(p *APLPrefix, msg []byte, off int) (int, error) {
 	if p.Negation {
 		n = 0x80
 	}
+	i := len(addr) - 1
```

Comment:
can you reference the RFC here?

Reply:
> can you reference the RFC here?

done, hopefully it addresses your intent.